clients/cli/src/main/resources/model_help.txt

```diff
@@ -0,0 +1,8 @@
+gcli model [details]
+
+Please set the metalake in the Gravitino configuration file or the environment variable before running any of these commands.
+
+Example commands
+
+Show model audit information
+gcli model details --name catalog_postgres.hr --audit
```

Comment:
This text doesn't seem to have an explicit format.
I know where we come from, but this can be improved.

Reply:
I'm not sure what you mean by "explicit format" in this context. This follows the format of the other help files.